core/ui/src/main/kotlin/com/google/samples/apps/nowinandroid/core/ui/NewsFeed.kt

```diff
@@ -62,7 +62,7 @@ fun LazyStaggeredGridScope.newsFeed(
                 contentType = { "newsFeedItem" },
             ) { userNewsResource ->
                 val context = LocalContext.current
-                val analyticsHelper = LocalAnalyticsHelper.current
+                val analyticsHelper = analyticsInstance(context)
```

Comment:
When the Composable is recomposed, accessing and retrieving from the Hilt dependency graph can incur costs. How about optimizing this by using `remember`? 🤔

Reply:
I agree that reducing cost
Thank you for your opinion

[commit](https://github.com/android/nowinandroid/pull/1472/commits/4fad93549a67035614f618ba7f647401992066ff)